vrs-annotator.wdl

```diff
@@ -20,6 +20,11 @@ workflow VRSAnnotator {
             compute_vrs_attributes = compute_vrs_attributes,
             genome_assembly = genome_assembly
     }
+
+    output {
+        File output_vcf = annotate.annotated_vcf
+        File output_vcf_index = annotate.annotated_vcf_index
```

Comment:
`output_vcf` and `output_vcf_index` are like default column names, right? Would it make sense to name them as `annotated_` instead of `output_`?

Reply:
no they're just supplied as variable names for the outputs, see image below. The name of the column you want to use is specified by the value supplied by the user. <img width="1632" alt="image" src="https://github.com/user-attachments/assets/7675fcbc-30c9-427c-81cf-4e3901014364">

See the [docs on writing to data table](https://support.terra.bio/hc/en-us/articles/4500420806299-Writing-workflow-outputs-to-the-data-table) and here's the [reference slack thread](https://gregorconsortium.slack.com/archives/C07RC3D79UZ/p1728592142728559?thread_ts=1728489060.766839&cid=C07RC3D79UZ).

If you haven't seen this before, could you launch a workflow to test this as part of review?

Reply:
Personally I'd like to keep it that way to create a distinction for input parameters, comparing an `input_vcf_path` path to an `output_vcf_name` rather than an `annotated_vcf_name`... and since it's called VRS Annotator, the output vcf is implied to be annotated. Lmk what you think.

Reply:
Works for me!